source/base/mpi.cc

```diff
@@ -581,10 +451,8 @@ namespace Utilities
       }();
 
       // If all processes report that they have unique destinations,
-      // then we can short-cut the process using a consensus algorithm (which
-      // is implemented only for the case of unique destinations, and also only
-      // for MPI 3 and later):
-#  if DEAL_II_MPI_VERSION_GTE(3, 0)
+      // then we can short-cut the process using a consensus algorithm:
+
```

Comment:
```suggestion
      // then we can short-cut the process using a consensus algorithm
      // (which is implemented only for the case of unique destinations):
```
I found this part of the parentheses still helpful